trino.yaml

```
@@ -26,7 +25,7 @@ environment:
       - go
       - jvmkill
       - maven
-      - openjdk-23
+      - openjdk-24
```

```suggestion
      - openjdk-24-default-jdk
```
nit: to be explicit with full jdk.